documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1737,17 +1672,176 @@ Typically, you configure the {prodname} SQL Server connector in a `.json` file u
 endif::community[]
 
 ifdef::product[]
+To deploy a {prodname} SQL Server connector, you add the connector files to Kafka Connect, create a custom container to run the connector, and then add connector configuration to your container.
+For details about deploying the {prodname} SQL Server connector, see the following topics:
+
+* xref:deploying-debezium-sql-server-connectors[]
+* xref:sql-server-connector-properties[]
+endif::product[]
+
+ifdef::product[]
+// Type: procedure
+// ModuleID: deploying-debezium-sql-server-connectors
+[[sql-server-deploying-a-connector]]
+=== Deploying {prodname} SQL Server connectors
```

Comment:
Plural intended here?

Reply:
It was, but I understand why it's giving you pause. Generally, we use plurals for procedural topic titles, because, although you might not ever create multiple connectors of a given type, it's a repeatable procedure, like if you had a procedure about "Putting on hats" or "Breaking windows." :)

Reply:
Ok, makes sense.